src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
```

Comment:
Please ensure that the `handle` method checks the result balance before saving it in the Storage and throws a RuntimeException if the balance is negative, as per the checklist handling Purchase operation.